### DISCUSSION TITLE
这周你做了什么？2022-06-17

### DISCUSSION THREAD

**Discussion by jiacai2050** [EmacsTalk/emacstalk.github.io#21] [Show and tell]
> 两三周没有更新 this-week，看看大家都在干啥呢

- 可以随便讨论本周你打算做什么与 Emacs 相关的事情，甚至可以寻求帮助或反馈。
- 请记住，什么都不做也没关系！
- Happy Hacking Emacs！

**Comment by beyondpie** [comment]
开始折腾一个在服务器端的emacs 配置。我目前的GUI配置在本地非常舒服，但是在服务器端启动异常慢。而如果打开一个vanilla TUI emacs，服务器端还是非常快的。这启发我，单独做一个基本原生的emacs 配置，如不加载evil，不加载magit， 不加载lsp等，尽可能就是原生，做最小的支持，在服务器端就足够。

**Comment by alaskasquirrel** [comment]
https://github.com/Lisp-Life/Lisp-China 资料集中 Common Lisp 完整不少，Emacs Lisp 期待分享参与。

**Reply by jiacai2050** [reply]
建议说下最近做的事情，感觉有些软广了。
